### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaushikharsh630**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ https://play.google.com/store/apps/details?id=com.varun.gbu_timetables
 
 - Support of both light and dark themes.
 
+-Soon aralrm feature is to come.
```
Comment (Varun-garg) [inline README.md]:
```suggestion
-Soon an alarm feature is to come.
```

File: README.md [hunk 2]
```diff
@@ -29,4 +31,4 @@ Please read our [contributing guidelines](CONTRIBUTING.md) to get started.
 
 ## Screenshots
 
-![Screenshots](Screenshots.jpg)
\ No newline at end of file
+![Screenshots](Screenshots.jpg)
```
Comment (Varun-garg) [inline README.md]: what is the diff here?

**Review by Varun-garg — COMMENTED** [review]
@kaushikharsh630  It would be pretty cool to have the feature. Looking forward to the change. + @amitkawasthi || @awasthi 

Please raise an issue wherever you need help and check out: https://github.com/Varun-garg/GBU_Timetables/blob/master/CONTRIBUTING.md

**Comment by kaushikharsh630** [comment]
There were many repos named "GBU time table" made the request to check
which one is active.
Don't worry about it I'm working on alarm feature and hopefully will be
able to do it.
By the way, is this the one on playstore?
Thankyou,
Harsh.n

**Comment by Varun-garg** [comment]
pretty big change, nice!

**Comment by Varun-garg** [comment]
@kaushikharsh630 awesome. I had put this one earlier in the play store, I think Amit sir will have a better idea to use which repo.

**Pull request merged**